insights/parsers/tests/test_krb5.py

```diff
@@ -0,0 +1,92 @@
+from insights.parsers import krb5
+from insights.tests import context_wrap
+
+
+KRB5CONFIG = """
+# Configuration snippets may be placed in this directory as well
+includedir /etc/krb5.conf.d/
+
+[logging]
+ default = FILE:/var/log/krb5libs.log
+ kdc = FILE:/var/log/krb5kdc.log
+ admin_server = FILE:/var/log/kadmind.log
+
+[realms]
+ dns_lookup_realm = false
+ auth_to_local = RULE:[1:$1@$0](.*@.*EXAMPLE.ORG)s/@.*//
+ default_ccache_name = KEYRING:persistent:%{uid}
+ default_ccache_name2 = KEYRING:%{uid}:persistent
+ EXAMPLE.COM = {
+  kdc = kerberos.example.com
+  admin_server = kerberos.example.com
+ }
+ 89EXAsdcsdMPLE.COM = {
+  kdc = ks2321serberos.example.com
+  admin_server = kc5433scderberos.example.com
+ }
+ ticket_lifetime = 24h
+[libdefaults]
+ dnsdsd = false
+ tilnvs = 24h
+ 89EXAsd = {
+  kdc = ks2321serberos.exampsdcsm
+  admin_server = kc5433scderberos.exasdscdm
+ }
+# renew_lifetime = 7d
+# forwardable = true
+# rdns = false
+""".strip()
+
+KRB5CONFIG2 = """
+# Configuration snippets may be placed in this directory as well
+""".strip()
+
+KRB5DCONFIG = """
+# Configuration snippets may be placed in this directory as well
+
+[loggingbb]
```

Comment:
Please do not make up examples.  Find a configuration file that has what you want to test by asking the IdM associates.